windows_bat/PokemonGo-Bot-Install.bat

```diff
@@ -100,39 +134,43 @@ if "%OS%" == "32-BIT" (
 
 
 :_choice
-set _ok=
-set /p _ok= Choose an option or press Enter to close: ||goto:eof
-if "%OS%" == "32-BIT" IF "%_ok%" == "1" SET CHOICE=32Bit&GOTO :32Bit
-if "%OS%" == "64-BIT" IF "%_ok%" == "1" SET CHOICE=64Bit&GOTO :64Bit
+SET _ok=
+SET /p _ok= Choose an option or press Enter to close: ||goto:eof
+IF "%OS%" == "32-BIT" IF "%_ok%" == "1" SET CHOICE=32Bit&GOTO :32Bit
+IF "%OS%" == "64-BIT" IF "%_ok%" == "1" SET CHOICE=64Bit&GOTO :64Bit
 IF "%_ok%" == "2" SET CHOICE=InstallBot&GOTO :InstallBot

```

Comment:
It installs the correct one. this line "reg Query "HKLM\Hardware\Description\System\CentralProcessor\0" | find /i "x86" > NUL && SET OS=32-BIT || SET OS=64-BIT" checks what OS the person has and the above give him only the choice of installing the OS he has